assets/js/common/autodownload.js

```diff
@@ -6,9 +6,15 @@
     'use strict';
 
     // Only do this on the autodownload page.
-    if ($('body').attr('id') == 'thunderbird-download') {
+    if ($('body').attr('id') === 'thunderbird-download') {
+        // IE11 safe check for determining if we've already downloaded Thunderbird
+        var lowercaseQuery = window.location.search.toLowerCase();
+        var isDownloaded = /([?|&]+downloaded=true)/.test(lowercaseQuery) === true;
```

Comment:
IE11 doesn't support SearchParams API, so a little regex here should ensure it doesn't break it.